.github/workflows/release-desktop.yml

```diff
@@ -40,13 +42,15 @@ jobs:
         uses: samuelmeuli/action-snapcraft@v1
         if: startsWith(matrix.os, 'ubuntu')
 
-      - name: Build/release Electron app
+      - name: Build & release Electron app
         uses: samuelmeuli/action-electron-builder@v1
-        if: ${{ startsWith(github.ref, 'refs/tags/v') }}
+        if: ${{ startsWith(github.ref, 'refs/tags/alephium-wallet@') }}
         with:
+          package_root: 'apps/desktop-wallet'
+          build_script_name: 'ci:build'
```

Comment:
Since `samuelmeuli/action-electron-builder` will execute the `build` script by default, we don't have a chance to run the `compile` script of the `shared` package. See later how `ci:build` fixes this.